docs/root/configuration/listeners/network_filters/wasm_filter.rst

```diff
@@ -18,20 +18,8 @@ Example configuration
 
 Example filter configuration:
 
-.. code-block:: yaml
-
-  name: envoy.filters.network.wasm
-  typed_config:
-    "@type": type.googleapis.com/envoy.extensions.filters.network.wasm.v3.Wasm
-    config:
-      config:
-        name: "my_plugin"
-        vm_config:
-          runtime: "envoy.wasm.runtime.v8"
-          code:
-            local:
-              filename: "/etc/envoy_filter_http_wasm_example.wasm"
-          allow_precompiled: true
+.. literalinclude:: _include/wasm-filter-proxy.yaml
+    :language: yaml
```

Comment:
probably we want to add some linenos, caption, etc - see other `literalincludes` for examples